Sources/PubNubChat/Storage/CoreData/CoreDataMigration.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jakub Guz on 6/22/22.
+//
+
+import Foundation
+import PubNub
+import CoreData
+
+class PayloadAlignmentMapper {
+  private let sourceModel: NSManagedObjectModel
+  private let destinationModel: NSManagedObjectModel
+  private let messageEntityClassName = "PubNubManagedMessage"
```

Comment:
It's a migration-specific code between two concrete `NSManagedObjectModel` versions. That's why I created something like `PayloadAlignmentMapper` to have the entire migration code from previous to current version. If other entities than `PubNubManagedMessage` were affected because of payload changes, I would put necessary code for them in this class as well. The only reason I extracted "PubNubManagedMessage" as a `let` variable was because I didn't want to use raw string anywhere. That could mislead you.

Anyway, having a common class for next migrations (I suppose that's what you mean) is a good option to consider. I will think about it. Let me know if you have any good idea.